book/README.md

```diff
@@ -1,53 +1,73 @@
-# TLG Catalog <a href='https://insightsengineering.github.io/tlg-catalog/'><img src="assets/img/logo.png" align="right" height="139" style="max-width: 100%; max-height: 139px;"/></a>
+# <a href='https://insightsengineering.github.io/tlg-catalog/'><img src="assets/img/logo.png" align="right" height="139" style="max-width: 100%; max-height: 139px;"/></a>
```

Comment:
I would keep the title

Reply:
If the title is kept, then it appears twice in a row on the site homepage. Do you think that's ok to leave?

i.e. Before change:
![Screenshot 2024-02-28 102211](https://github.com/insightsengineering/tlg-catalog/assets/59304861/e31e94cb-d251-4f52-b5cc-b85bf0d1f32b)

After change:
![Screenshot 2024-02-28 102239](https://github.com/insightsengineering/tlg-catalog/assets/59304861/7f215ae0-d649-4f45-b8b3-0492b349b86e)

Reply:
with the enriched review I get 
<img width="1154" alt="Screenshot 2024-02-29 094418" src="https://github.com/insightsengineering/tlg-catalog/assets/11279768/80eddb60-fa06-400b-9933-22f8b33a469f">